src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,271 @@
+
+- A Revolutionary speaking a codeword through a headset radio deals **0.2**x LHP damage. (Note that crew passively heal LHP, as described in the Deconversion section)
+- The first time a Revolutionary uses the Announcement console to send a message with a codeword, it deals **1**x LHP damage, without being divided among the people hearing it. This is a one-time trigger.
+
+### Deconversion
+
+Deconversion is the main way for crew to fight back against the revolution. It should be fairly easy when targeted at isolated Revolutionaries but require more work when Revolutionaries are together. It should also not happen by accident; crew should consciously attempt it to be successful.
+
+Deconversion puts you fully on the crew side. This does fully include the ability to sell out your fellow Revolutionaries by remembering their names. Headrevs therefore have a choice to act more stealthily disguise themselves and blend in with the Revolution, or assign another Revolutionary (possibly their first convert) to act as the revolution's leader to throw off potential deconvert sell-outs. 
+
+Methods of deconversion include:
+  - Harm baton, heals **10** LHP per hit
+  - Being in cuffs heals **0.5** LHP / second
+  - Wearing an electro pack heals **0.5** LHP / second
+
+Loyal crew are also able to passively heal LHP at **0.1** LHP / second, though as this does not happen for Revolutionaries it can not be used for deconversion. This passive heal is disabled for **10** seconds upon taking LHP damage.
+
+A deconverted Revolutionary can be reconverted again. If a Headrev is deconverted (>= 0 LHP) they lose their antag status, and being converted to the Revolution makes you a normal Revolutionary. 
+
+### Death & LHP Damage
+
+Authoritarian regimes killing of protesting civilians causing further unrest, or freedom fighters getting a shock when they first murder someone, are concepts that are closely tied to a revolution. They are also ideas that would be suitable mechanically to combat murder sprees from either side during a Revolution round. The following section is only relevant for crew and Revs; non-crew characters (pets, mobs) and other antags (traitors, Nukies) do not interact with the following mechanic.
+
+If a Revolutionary *or crewmember* dies by the hand of another crewmember (being dealt the damage that put them in a critical state), they will deal **15** non-split LHP in a **8** tile line-of-sight radius around their body. While remaining dead they will also provide a passive **-0.1** LHP / second in a **4** tile line-of-sight radius. This works in the same way but in reverse if a Revolutionary performs the kill, healing LHP upon death when killed by a Revolutionary/Crewmember. The initial burst also has a **120** second cooldown, to avoid repeatedly reviving and killing the same person twice to trigger the effect. This means both sides are discouraged from indiscriminately killing, favoring limited killing and kidnapping. 
```

Comment:
This is sort of a complex idea so I'll try and make sure I have it straight: crewmembers take LHP when they see shocking things happen, but Revs will gain LHP when this happens. This can cause flips, and if it's Revs->Crew then it's a 1s stun?

Reply:
Kill the other side: bad for your LHP, good for the other side's LHP
Kill your own side: bad for your LHP, good for the other side's LHP